src/index.js

```diff
@@ -1,42 +1,2 @@
 import './style.css';
-import deleteImage from './menu.svg';
-
-const todoTasks = [
-  {
-    description: 'wash the dishes',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'complete todo list project',
-    completed: true,
-    index: 2,
-  },
-];
-
-// var elem = document. createElement("hr");
-// elem. setAttribute("width", "100px");
-// document. body. appendChild(elem);
-
-function populate() {
-  const listTodo = document.querySelector('.todo-list');
-  todoTasks.forEach((todo) => {
-    const li = document.createElement('li');
-    const div = document.createElement('div');
-    const checkBox = document.createElement('input');
-    const menuIcon = document.createElement('img');
-    const line = document.createElement('hr');
-    const p = document.createElement('p');
-    p.innerHTML = todo.description;
-    checkBox.setAttribute('type', 'checkbox');
-    menuIcon.setAttribute('src', `${deleteImage}`);
-    line.setAttribute('width', '100%');
-    li.appendChild(checkBox);
-    div.appendChild(p);
-    div.appendChild(menuIcon);
-    li.appendChild(div);
-    listTodo.appendChild(line);
-    listTodo.appendChild(li);
-  });
-}
-populate();
+import './todo.js';
```

Comment:
Kindly the application is not showing in the browser. try to fix it . this is the message error am getting `The resource from “http://localhost:8080/index.js” was blocked due to MIME type (“text/html”) mismatch (X-Content-Type-Options: nosniff).`
![Screenshot from 2022-04-30 16-23-45](https://user-images.githubusercontent.com/77842237/166107599-031b5aeb-fcd9-4e5a-a027-3b08e2c40917.png)

Reply:
Kindly read the README file, the program runs with _npm start_